src/app/hero-search/hero-search.component.ts

```diff
@@ -0,0 +1,33 @@
+import { HeroService } from './../hero.service';
+import { Hero } from './../hero';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/infinity-arc/Kobus-tour-of-heroes/pullRequest?prid=7838883)
```suggestion
import { Hero } from "./../hero";
```